HelpMyStreetFE/HelpMyStreetFE/Controllers/HomeController.cs

```diff
@@ -34,6 +37,7 @@ public async Task<IActionResult> Index()
             var model = new HomeViewModel
             {
                 isLoggedIn = ((HttpContext.User != null) && HttpContext.User.Identity.IsAuthenticated),
+                NewsTickerMessages = await _newsTickersService.GetNewsTickerMessages(null)
```

Comment:
If you add CancellationToken cancellationToken as a parameter to this method, it'll be populated automatically by MVC, and you can then pass it through to the NewsTickersService.  (See comment below...)